Extensions/IISWebAppDeploy/Src/Tasks/IISWebAppMgmt/IISWebAppMgmtV3/task.json

```diff
@@ -0,0 +1,740 @@
+{
+  "id": "0F5CD14F-3C01-4D5C-8F7A-EB96C5738DCC",
+  "name": "IISWebAppMgmt",
+  "friendlyName": "WinRM - IIS Web App Management",
+  "description": "Connect via WinRM, to create or update a Website, Web App, Virtual Directories, and Application Pool locally on IIS",
+  "helpMarkDown": "[More Information](http://aka.ms/IISMgmt)",
+  "category": "Deploy",
+  "visibility": [
+    "Build",
+    "Release"
+  ],
+  "runsOn": [
+      "Agent"
+  ],
+  "author": "Microsoft Corporation",
+  "version": {
+    "Major": 3,
+    "Minor": 0,
+    "Patch": 0
+  },
+  "demands": [
+  ],
+  "releaseNotes": "What's new in Version 3.0: <br/>&nbsp;&nbsp; Supported the task on PS3 Handler",
```

Comment:
Generally, we put changes logs for the extension instead of the individual task.

Reply:
I will do that also